jmix-gridexport/gridexport-flowui/src/main/java/io/jmix/gridexportflowui/exporter/EntityExporter.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2023 Haulmont.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.jmix.gridexportflowui.exporter;
+
+/**
+ * Entity exporter which is used by {@link AbstractAllRecordsExporter} to export entities one by one
```

Comment:
1. This exporter can be used not only by `AbstractAllRecordsExporter`, so JavaDoc must be more neutral. 
2. Do we really need it? It seems that it can be replaced with standard `java.util.function.Predicate` that accepts some `XxxContext` object. We try not create own function interfaces if they can be replaced with standard ones. Also, a good practice replacing several input parameters with a single object, that allows to add/deprecate these parameters.